python/kserve/requirements.txt

```diff
@@ -16,7 +16,7 @@ cloudevents>=1.2.0
 avro>=1.10.1
```

Comment:
also tried to update python to 3.9-slim in [the Dockerfile](https://github.com/mesosphere/kfserving/blob/7810c47d614700f1e9a88450ba99dc781a5c4156/python/storage-initializer.Dockerfile#L1), but that did not result in less critical CVEs